src/config/get-config.ts

```diff
@@ -81,6 +81,15 @@ export interface Config {
    * is set to `false`, then the feature will be available in all channels
    */
   AUTO_LINK_CHANNEL: string | false;
+  /**
+   * The role to assign to users who wish to be notified of new live streams.
+   */
+  STREAM_NOTIF_ROLE: string;
```

Comment:
Change to `STREAM_NOTIFY_ROLE`, not really worth it to save 1 character, or maybe its a typo :shrug: 

```suggestion
  STREAM_NOTIFY_ROLE: string;
```

Reply:
Haha `NOTIFY` is good - I was thinking of `NOTIFICAITONS` and didn't want to type that out